tests/integration_tests.rs

```diff
@@ -1012,3 +1033,62 @@ async fn test_pod_mounts_with_pvc() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+const RESOURCE_NAME: &str = "example.com/gpu";
+const RESOURCE_ENDPOINT: &str = "gpu-device-plugin.sock";
+
+async fn create_device_plugin_resource_pod(
+    client: kube::Client,
+    pods: &Api<Pod>,
+    resource_manager: &mut TestResourceManager,
+) -> anyhow::Result<()> {
+    use k8s_openapi::apimachinery::pkg::api::resource::Quantity;
+    let containers = vec![
+        WasmerciserContainerSpec::named("device-plugin-test").with_args(&[
+            "write(lit:watermelon)to(stm:stdout)",
```

Comment:
If volume testing was in place, this would ideally be:
```
WasmerciserContainerSpec::named("device-plugin-test").with_args(&[
    "write(lit:watermelon)to(file:/brb/general.txt)",
    "read(file:/brb/general.txt)to(var:myfile)",
    "write(var:myfile)to(stm:stdout)",
    "assert_exists(env:DEVICE_PLUGIN_VAR)",
]),
```

Reply:
Do we need this `write` line right now if we aren't using it?

Reply:
Good point. Removed in ce2fcb4531488cd7f4fa7086b9add02d36b21c5e